src/hoki.js

```diff
@@ -68,33 +50,24 @@ function observer(event, callback) {
         throw new TypeError('Callback must be a function');
     }
 
-    if (!events[event]) {
-        return;
-    }
-
-    events[event].push(callback);
+    if (events[event]) events[event].push(callback);
 }
 
-// Dispatch events with or without data
+// Dispatch events
 function dispatcher(event, data) {
-    if (!isString(event)) {
-        throw new TypeError('Event must be a string');
-    }
-
-    if (!events[event]) {
-        return;
-    }
-
-    events[event].forEach(callback => (data ? callback(data) : callback()));
+    if (!isString(event)) throw new TypeError('Event must be a string');
+    
+    // call every callback in event list
+    if (events[event]) events[event].forEach(cb => cb(data));
 }
 
 // Return events available
-const list = () => Object.keys(events);
+const events = () => Object.keys(events);
```

Comment:
there is a reason I renamed it to list. Because I renamed the event container to events, so the const is already declared.
The events object could be renamed to something else, to keep the code consistent.

Reply:
oh my bad, didn't notice sorry just change it back then.